Calculator/Calculator/Extension/String+.swift

```diff
@@ -48,6 +48,30 @@ extension String {
         }
     }
     
+    func convertToExponent() -> String {
+        guard self.count > 20 else {
+            return self
+        }
+        let splittedNumber = self.components(separatedBy: NameSpace.dot)
+        let eCount = splittedNumber[0].count - 1
+        let stringToBeRounded: String
+        if splittedNumber.count == 1 {
+            stringToBeRounded = splittedNumber[0]
+        } else {
+            stringToBeRounded = splittedNumber[0] + splittedNumber[1]
+        }
+        var newString = NameSpace.emptyString
+        for number in stringToBeRounded {
+            newString += String(number)
+            if newString.count == 17 {
+                break
+            }
+        }
+        newString.insert(Character(NameSpace.dot), at: newString.index(newString.startIndex, offsetBy: 1))
+        newString += "e" + String(eCount)
+        return newString
+    }
```

Comment:
[P3] 변수 선언, 조건문, 반복문 사이에 개행을 주면 가독성이 더 좋아질 것 같아요. 
추가적으로 수를 String으로 전환할 때 String Interpolation을 사용하는 방법도 있습니다.

```suggestion
    func convertToExponent() -> String {
        guard self.count > 20 else {
            return self
        }

        let splittedNumber = self.components(separatedBy: NameSpace.dot)
        let eCount = splittedNumber[0].count - 1
        var stringToBeRounded = splittedNumber[0]

        if splittedNumber.count != 1 {
            stringToBeRounded += splittedNumber[1]
        }

        var newString = NameSpace.emptyString

        for number in stringToBeRounded where newString.count < 17 {
            newString += "\(number)"
        }

        let index = newString.index(newString.startIndex, offsetBy: 1)
        newString.insert(Character(NameSpace.dot), at: index)
        newString += "e\(eCount)"

        return newString
    }
```

Reply:
변수/상수, 조건문, 반복문을 기준으로 전체적으로 개행 진행하였습니다.